examples/01-basic/11-placeholder/README.md

```diff
@@ -0,0 +1,3 @@
+# Placeholder
+
+In this example, we pass in a custom placeholder to change the placeholder text of the editor.
```

Comment:
> In this example, we customize the dictionary to pass in custom placeholder texts. This way, we change the text of an empty block (and empty heading block) in the editor.

Reply:
Also, follow the default template and add a link to relevant docs, same as on https://www.blocknotejs.org/examples/basic/localization